### PR TITLE
#3684 [Manager] remove the delete button in the private topic detail …

### PR DESCRIPTION
**Pull request by eechih**
…page

### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3684 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/theme/muiTheme.js [hunk 1]
```diff
@@ -58,22 +58,6 @@ const MuiTheme = createMuiTheme({
...
-      500: '#ffc107',
-      600: '#ffb300',
-      700: '#ffa000',
-      800: '#ff8f00',
-      900: '#ff6f00',
-      main: '#ffc107',
-      light: '#ffd54f',
-      dark: '#ffa000',
-      contrastText: '#FFFFFF',
-    },
```
Comment (eechih) [inline ohara-manager/client/src/theme/muiTheme.js]: 某些 mui components (例如 Button) 不支援 color='error'，因此將 secondary 改回預設值（紅色）
/cc @konekoya @saivirtue

Comment (saivirtue) [reply ohara-manager/client/src/theme/muiTheme.js]: 意思是其他component都不用動嗎?

Comment (eechih) [reply ohara-manager/client/src/theme/muiTheme.js]: Yep

File: ohara-manager/client/src/context/dialog/dialogNames.js [hunk 2]
```diff
@@ -14,14 +14,14 @@
...
+export const ADD_TOPIC_DIALOG = 'addTopic';
+export const VIEW_TOPIC_DIALOG = 'viewTopic';
+export const LIST_NODE_DIALOG = 'listNode';
+export const VIEW_NODE_DIALOG = 'viewNode';
+export const EDIT_NODE_DIALOG = 'editNode';
+export const ADD_NODE_DIALOG = 'addNode';
+export const DEV_TOOL_DIALOG = 'devTool';
+export const GRAPH_SETTING_DIALOG = 'graphSetting';
+export const LIST_WORKSPACES_DIALOG = 'listWorkspace';
+export const ADD_PLUGIN_DIALOG = 'addPlugin';
```
Comment (konekoya) [inline ohara-manager/client/src/context/dialog/dialogNames.js]: 👍

File: ohara-manager/client/src/components/Topic/TopicChip.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        label={isShared ? 'Shared' : 'Private'}
+        color={isShared ? 'primary' : 'secondary'}
+        className={isShared ? 'shared' : 'private'}
+      />
+    </Wrapper>
+  );
+}
+
+TopicChip.propTypes = {
+  isShared: PropTypes.bool,
```
Comment (konekoya) [inline ohara-manager/client/src/components/Topic/TopicChip.js]: Should we provide a default value for this if the value is not required :)

**Pull request merged**